### PR TITLE
KNL-1439 When a the cm provider updates a realm post an event

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

**Comment by juanjmerono** [comment]
LGTM

File: kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/DbAuthzGroupService.java [hunk 1]
```diff
@@ -685,10 +685,7 @@ public void run() {
 					long time = 0;
 					long start = System.currentTimeMillis();
 					try {
-						// only remove from the cache if the realm was updated during the refresh
-						if (((DbStorage) m_storage).refreshAuthzGroupInternal((BaseAuthzGroup) azGroup)) {
-							m_realmRoleGRCache.remove(azGroupId);
```
Comment (steveswinsburg) [inline kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/DbAuthzGroupService.java]: has this remove line been lost?

**Comment by juanjmerono** [comment]
It works right @ern !!

File: kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/DbAuthzGroupService.java (hunk 1 above)
Comment (ern) [inline kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/DbAuthzGroupService.java]: Nope as that only occurred in a thread processing the queue, it now fires an event which will then do the invalidation.

**Pull request merged**